### PR TITLE
Example for list of value

### PR DESCRIPTION
**Pull request by namnguyen20999**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

Adding code example for list of value for Taipy extensions

- [ ] Does this solution meet the acceptance criteria of the related issue?
- [ ] Is the related issue checklist completed?
- [ ] Does this PR adds unit tests for the developed code? If not, why?
- [ ] End-to-End tests have been added or updated?
- [X] Was the documentation updated, or a dedicated issue for documentation created? (If applicable)
- [ ] Is the release notes updated? (If applicable)

### CODE REVIEW

**Review by FredLL-Avaiga — DISMISSED** [review]
A bit simplistic
we could add an id and icon to the languages list ?

**Comment by namnguyen20999** [comment]
> A bit simplistic we could add an id and icon to the languages list ?

id & icon added

File: doc/gui/extension/example_library/front-end/src/ListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        alignItems: "center",
+    },
+    listItemImage: {
+        marginRight: "8px",
+        width: "1em",
+        height: "1em",
+    },
+};
+
+const ListItem: React.FC<ToDoListProps> = ({ lov, defaultLov = "", sort }) => {
```
Comment (FredLL-Avaiga) [inline doc/gui/extension/example_library/front-end/src/ListItem.tsx]: this is not a listItem as it shows full list ?

File: doc/gui/extension/cpp.png
Comment (FredLL-Avaiga) [inline doc/gui/extension/cpp.png:1]: group the images in a folder ?

File: doc/gui/extension/example_library/front-end/src/ListOfLanguages.tsx
Comment (FredLL-Avaiga) [inline doc/gui/extension/example_library/front-end/src/ListOfLanguages.tsx:1]: not used ?

**Review by FredLL-Avaiga — DISMISSED** [review]

File: doc/gui/extension/example_library/front-end/src/ListItem.tsx (hunk 1 above)
Comment (namnguyen20999) [reply doc/gui/extension/example_library/front-end/src/ListItem.tsx]: Name changed

File: doc/gui/extension/cpp.png
Comment (namnguyen20999) [reply doc/gui/extension/cpp.png:1]: grouped

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**